flexget/utils/imdb.py

```diff
@@ -166,7 +168,15 @@ def search(self, name):
             movie = {}
             additional = re.findall(r'\((.*?)\)', row.text)
             if len(additional) > 0:
-                movie['year'] = additional[-1]
+                try:
+                    int(additional[-1])
```

Comment:
I think either `isdigit()` or regex is better. Since you want it to be a four digit year (let's not write too general code here), you can use regex to check that it is indeed four digits.